modules/effects/arrow-effects-reactor/src/test/kotlin/arrow/effects/FluxKTests.kt

```diff
@@ -1,58 +1,52 @@
 package arrow.effects
 
+import arrow.effects.typeclasses.ExitCase
 import arrow.test.UnitSpec
-import arrow.test.laws.AsyncLaws
-import arrow.test.laws.FoldableLaws
-import arrow.test.laws.TraverseLaws
 import arrow.typeclasses.Eq
-import arrow.typeclasses.binding
-import arrow.typeclasses.bindingCatch
 import io.kotlintest.KTestJUnitRunner
-import io.kotlintest.matchers.shouldNotBe
 import org.hamcrest.CoreMatchers.not
 import org.hamcrest.CoreMatchers.startsWith
 import org.hamcrest.MatcherAssert.assertThat
 import org.junit.runner.RunWith
 import reactor.core.publisher.Flux
-import reactor.core.scheduler.Schedulers
 import reactor.test.test
 import java.time.Duration
 
 @RunWith(KTestJUnitRunner::class)
 class FluxKTest : UnitSpec() {
 
   fun <T> assertThreadNot(flux: Flux<T>, name: String): Flux<T> =
-      flux.doOnNext { assertThat(Thread.currentThread().name, not(startsWith(name))) }
+    flux.doOnNext { assertThat(Thread.currentThread().name, not(startsWith(name))) }
 
   fun <T> EQ(): Eq<FluxKOf<T>> = object : Eq<FluxKOf<T>> {
     override fun FluxKOf<T>.eqv(b: FluxKOf<T>): Boolean =
-        try {
-          this.value().blockFirst() == b.value().blockFirst()
-        } catch (throwable: Throwable) {
-          val errA = try {
-            this.value().blockFirst()
-            throw IllegalArgumentException()
-          } catch (err: Throwable) {
-            err
-          }
-
-          val errB = try {
-            b.value().blockFirst()
-            throw IllegalStateException()
-          } catch (err: Throwable) {
-            err
-          }
-
-          errA == errB
+      try {
+        this.value().blockFirst() == b.value().blockFirst()
+      } catch (throwable: Throwable) {
+        val errA = try {
+          this.value().blockFirst()
+          throw IllegalArgumentException()
+        } catch (err: Throwable) {
+          err
         }
+
+        val errB = try {
+          b.value().blockFirst()
+          throw IllegalStateException()
+        } catch (err: Throwable) {
+          err
+        }
+
+        errA == errB
+      }
   }
 
   init {
-
+/*
```

Comment:
Removed for testing?

Reply:
Oops sorry, commented this for reducing noise amount when testing something else. Uncommented now 👍